$OpenBSD: patch-messageviewer_grantleefilters_CMakeLists_txt,v 1.1 2015/03/30 17:11:45 zhuk Exp $
Unbreak build (exported symbols from MOC present in two object files).
--- messageviewer/grantleefilters/CMakeLists.txt.orig	Thu Mar 26 12:36:31 2015
+++ messageviewer/grantleefilters/CMakeLists.txt	Thu Mar 26 12:36:51 2015
@@ -10,6 +10,7 @@ INCLUDE_DIRECTORIES( ${PROJECT_BINARY_DIR} ${KDE4_INCL
 # is not found, and will not tell the user about the missing REQUIRED package.
 if(Grantlee_FOUND)
   grantlee_add_plugin(grantlee_messageheaderfilters messageheadergrantleefilters FILTERS messageheaderfilter )
+  set_target_properties(grantlee_messageheaderfilters PROPERTIES AUTOMOC No)
   TARGET_LINK_LIBRARIES(grantlee_messageheaderfilters messagecore)
 
 ########### install files ###############
